alpine/BumponTailInstabilityManager.h

```diff
@@ -0,0 +1,613 @@
+#ifndef IPPL_BUMPON_TAIL_INSTABILITY_MANAGER_H
```

Comment:
Missing licensing information at top of file.

Reply:
@Arc676 After switching to the BSD-3 license we stopped putting licensing info at top of every file. Rather there is only one file at the root level of the repo. I assume you were asking about that only.